recipes/bitserializer-pugixml/all/conanfile.py

```diff
@@ -0,0 +1,39 @@
+from conans import ConanFile, tools
+import os
+
+
+class BitserializerConan(ConanFile):
+    name = "bitserializer-pugixml"
+    description = "C++ 17 library for serialization, module for support XML (implementation based on the PugiXml library)"
+    topics = ("serialization", "xml")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://bitbucket.org/Pavel_Kisliak/bitserializer"
+    license = "MIT"
+    settings = ("compiler",)
+    no_copy_source = True
+    requires = ("bitserializer/0.9", "pugixml/1.10@bincrafters/stable")
```

Comment:
```suggestion
    requires = ("bitserializer/0.9", "pugixml/1.10@bincrafters/stable")
```

Bincrafters recipes are not accepted here in CCI. pugixml need to be ported first

Reply:
Understood, thanks!